---
layout: default
---

{% include header.md %}

## Program

### Tuesday December 6, 2016

08:30 - 09:00
: Registration

Session I, chairman: _Florian Hahne_

09:00 - 10:00
: [TALK](slides/TALK_Martin_Morgan.pdf) _Martin Morgan_: "Package review process"

10:00 - 10:15
: [FLT01](slides/FLT01_Andrzej_Oles.html) _Andrzej Oles_: "Authoring R Markdown documents with BiocStyle"

10:15 - 10:30
: [FLT02](slides/FLT02_Jitao_David_Zhang.pptx) _Jitao David Zhang_: "ribios, a suite of R packages dedicated to high-performance and pipelineable bioinformatics analysis"

10:30 - 10:45
: [FLT03](slides/FLT03_Mike_Smith.pdf) _Mike Smith_: "BiocWorkflowTools package"

10:45 - 11:05
: [FLT04](slides/FLT04_Wolfgang_Huber.pdf) _Wolfgang Huber_: "More powerful FDR control with data-driven hypothesis weighting"

11:05 - 11:30
: Coffee break

Session II, chairman: _Mark Robinson_

11:30 - 11:45
: [FLT05](slides/FLT05_Federico_Marini.html) _Federico Marini_: "Development of Applications for Interactive and Reproducible Research: the pcaExplorer Case Study"

11:45 - 12:00
: [FLT06](slides/FLT06_Phil_Cheng.pdf) _Phil Cheng_: "TCGA browser package"

12:00 - 12:15
: [FLT07](slides/FLT07_Nuno_Agostinho.pdf) _Nuno Agostinho_: "Shiny application for the integrated analysis of alternative splicing from large transcriptomic datasets (TCGA)"

12:15 - 12:30
: [FLT08](slides/FLT08_Carles_Hernandez-Ferrer.pdf) _Carles Hernandez-Ferrer_: "rexposome: A Bioconductor package for characterizing multiple environmental factors and its association with different omics biomarkers and disease"

12:30 - 12:45
: [FLT09](slides/FLT09_Davis_McCarthy.pdf) _Davis McCarthy_: "scater: pre-processing, quality control, normalisation and visualisation for single-cell RNA-seq data"

12:45 - 14:15
: Lunch

Session III, chairman: _Charlotte Soneson_

14:15 - 14:30
: [FLT10](slides/FLT10_Laurent_Gatto.pdf) _Laurent Gatto_: "MSnbase2 - disk access is the limit"

14:30 - 14:45
: [FLT11](slides/FLT11_Witold_Wolski.pdf) _Witold Wolski_: "bibliospec: Reading Mass Spectrometric Search Results"

14:45 - 15:00
: FLT12 _Cancelled_

15:00 - 16:00
: KEYNOTE I [Anne-Laure Boulesteix](http://www.ibe.med.uni-muenchen.de/organisation/mitarbeiter/020_professuren/boulesteix/): "Towards evidence-based computational statistics: lessons from clinical research on the role and design of real-data benchmark studies" <br />The goal of medical research is to develop interventions that are in some sense superior, with respect to patient outcome, to interventions currently in use. Similarly, the goal of research in methodological computational statistics is to develop data analysis tools that are themselves superior to the existing tools. The methodology of the evaluation of medical interventions continues to be discussed extensively in the literature and it is now well accepted that medicine should be at least partly "evidence based". Although we statisticians are convinced of the importance of unbiased, well-thought-out study designs and evidence-based approaches in the context of clinical research, we tend to ignore these principles when designing our own studies for evaluating statistical methods in the context of our methodological research. In this talk, I draw an analogy between clinical trials and real-data-based benchmarking experiments in methodological statistical science, with datasets playing the role of patients and methods playing the role of medical interventions, as suggested in the technical report by Boulesteix, Wilson & Hapfelmeier (2016, [https://epub.ub.uni-muenchen.de/29889/](https://epub.ub.uni-muenchen.de/29889/) ). Through this analogy, I discuss the concept of "evidence-based" statistical research, its limitations and its impact on the design and interpretation of real-data-based benchmark experiments as well as potential implications for Bioconductor development.

16:00 - 16:30
: Coffee Break

Session IV, chairman: _Michael Stadler_

16:30 - 16:45
: [FLT13](slides/FLT13_Raffaele_Calogero.pdf) _Raffaele Calogero_: "PAMR-SC: Subpopulation classification in single-cell sequencing"

16:45 - 17:00
: [FLT14](slides/FLT14_Helena_Crowell.pdf) _Helena Crowell_: "CATALYST package (Cytometry dATa AnaLYSis Tools)"

17:00 - 18:00
: KEYNOTE II [Niko Beerenwinkel](https://www.bsse.ethz.ch/department/people/detail-person.html?persid=149417): "Modeling cancer evolution from genomic data" <br /> Cancer progression is an evolutionary process characterized by the accumulation of mutations and responsible for tumor growth, clinical progression, and drug resistance development. Evolutionary theory can be used to describe the dynamics of tumor cell populations and to make inference about the evolutionary history of a tumor from molecular profiling data. We present recent approaches to modeling the evolution of cancer, including population genetics models of tumorigenesis, phylogenetic methods of intra-tumor subclonal diversity, and probabilistic graphical models of tumor progression, and we discuss their relevance for personalized medicine in general and for precision oncology in particular.

19:00 -
: Dinner in Restaurant LÃ¶wenzorn


### Wednesday December 7, 2016

Session V, chairman: _Vincent Carey_

09:00 - 09:15
: [FLT15](slides/FLT15_Andrea_Komljenovic.pdf) _Andrea Komljenovic_: "BgeeDB package"

09:15 - 09:30
: [FLT16](slides/FLT16_Julien_Roux.pdf) _Julien Roux_: "BgeeDB package"

09:30 - 09:45
: FLT17 _Michal Okoniewski_: "Connecting to genomic data in Parquet using SparkR"

09:45 - 10:00
: [FLT18](slides/FLT18_Johannes_Rainer.pdf) _Johannes Rainer_: "Extending ensembldb: MySQL backend and protein annotations"

10:00 - 10:20
: [FLT19](slides/FLT19_Parham_Solaimani.pdf) _Parham Solaimani_: "Supporting Bioconductor packages in Renjin"

10:20 - 10:30
: [FLT20](http://www.youtube.com/watch?v=ZAbaXAQpZPE) _Levi Waldron_: "curatedMetagenomicData"

10:30 - 10:45
: [FLT21](slides/FLT21_Vince_Carey.pdf) _Vince Carey_: "Recent developments in MultiAssayExperiment"

10:45 - 11:00
: Discussion Session

11:00 - 11:30
: Coffee Break

Session VI, chairman: _Wolfgang Huber_

11:30 - 13:00
: Discussion Session

13:00 - 14:00
: Lunch

14:00 - 15:30
: Wrap up
